modules/policy/engine/src/main/java/org/eclipse/sw360/antenna/policy/engine/RuleSeverity.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.policy.engine;
+
+/**
+ * The severity levels of policy violations, this enables the user of the engine to react
+ * appropriately on a policy violation
+ */
+public enum RuleSeverity {
+    INFO,
+    WARN,
+    FAIL;
```

Comment:
Nit: IMO "FAIL" is not a severity, it's an action you may take *based on* the severity of an issue. (Or did you mean to says "FAILURE"?)

FYI, in ORT [we have](https://github.com/heremaps/oss-review-toolkit/blob/master/model/src/main/kotlin/Severity.kt) "ERROR", "WARNING" and "HINT".

Reply:
I basically took them over from the Antenna concept. There the severity is something that impacts the build, so FAIL means, fail the build. But I understand your issue there, it is the decision of the workflow engine to decide what to do in case of an error.

Reply:
Switch FAIL to ERROR